src/wallet/bdb.h

```diff
@@ -208,10 +208,10 @@ class BerkeleyBatch
     };
 
 private:
-    bool ReadKey(CDataStream& key, CDataStream& value);
-    bool WriteKey(CDataStream& key, CDataStream& value, bool overwrite=true);
-    bool EraseKey(CDataStream& key);
-    bool HasKey(CDataStream& key);
+    bool ReadKey(Span<char> key, CDataStream& value);
```

Comment:
These Spans can be const char ones. That also has the advantage of supporting automatic conversion from temporaries.

Reply:
`SafeDbt` is used to assign `0` to the `char`s, which wouldn't work with `const char`